docs/basic-features/data-fetching.md

```diff
@@ -94,6 +94,27 @@ export const getStaticProps: GetStaticProps = async context => {
 }
 ```
 
+If you want to get inferred typings for your props, you can use `InferredStaticProps<typeof getStaticProps>` like this:
+
+```ts
+import { GetStaticProps, InferredStaticProps } from 'next'
+
+export const getStaticProps: GetStaticProps = async context => {
+  // ...
+  return {
+    props: {
+      posts: await prismaClient.post.fetchMany(),
```

Comment:
Let's use `fetch` here too, basically follow the same pattern used in the other samples in this section.